### ISSUE TITLE
Task: Feedback additional questions not read out by screen reader

### ISSUE THREAD

**Issue by DeanWillicombe82** [ministryofjustice/data-platform#166]
### Task Description

The feedback 'Yes/No' question, when the button is selected, brings up more questions however these are not announced to a screen reader user. They may not be aware they need to answer any other questions. 

Suggested Fix: Use an aria-live for anything that changes dynamically on the page. Or divert the user to a new tab/page to answer these questions.

If keeping on the same page, once the user answers the question perhaps it can contain a header, 'Tell us more why this page was useful' to give it more definition. The screen reader/focus could land on here.

### Acceptance Criteria

Fix meets 4.1.3 Status Messages WCAG criteria

### Steps to Address

UX designer to suggest fix and create implementation ticket as sub issue for devs to work on.

### Useful Contacts

matt.proctor-leake@justice.gov.uk

### Links & References

[Link to review document](https://justiceuk.sharepoint.com/:x:/r/sites/MoJAccessibilityTeam/Shared%20Documents/01.%20Consultancy/Consultancy%20requests/02-02-2026-Find%20MOJ%20Data-Dean%20Willicombe/Find%20MOJ%20Data%20accessibility%20review.xlsx?d=w478bd709058d458c94e7885c660b984c&csf=1&web=1&e=0HlU9H)
